iree_tests/pytorch/models/sdxl-prompt-encoder-tank/test_cases.json

```diff
@@ -0,0 +1,29 @@
+{
+  "test_cases": [
+    {
+      "name": "splats",
+      "runtime_flagfile": "splat_data_flags.txt",
+      "remote_file_groups": []
+    },
+    {
+      "name": "real_weights",
+      "runtime_flagfile": "real_weights_data_flags.txt",
+      "remote_file_groups": [
+        {
+          "azure_account_url": "https://sharkpublic.blob.core.windows.net",
+          "azure_container_name": "sharkpublic",
+          "azure_base_blob_name": "sai/sdxl-prompt-encoder/",
+          "files": [
+            "inference_input.0.bin",
+            "inference_input.1.bin",
+            "inference_input.2.bin",
+            "inference_input.3.bin",
+            "inference_output.0.bin",
+            "inference_output.1.bin",
+            "real_weights.irpa"
+          ]
+        }
+      ]
+    }
+  ]
+}
```

Comment:
Seeing more of these files, I'm still thinking about how to keep them easy to update. Might refactor to separate JSON files like

test_case_splats.json:
```json
{
  "name": "splats",
  "runtime_flagfile": "splat_data_flags.txt",
  "remote_files": []
}
```

test_case_real_weights.json:
```json
{
  "name": "real_weights",
  "runtime_flagfile": "real_weights_data_flags.txt",
  "remote_files": [
    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sdxl-prompt-encoder/inference_input.0.bin",
    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sdxl-prompt-encoder/inference_input.1.bin",
    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sdxl-prompt-encoder/inference_input.2.bin",
    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sdxl-prompt-encoder/inference_input.3.bin",
    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sdxl-prompt-encoder/inference_output.0.bin",
    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sdxl-prompt-encoder/inference_output.1.bin",
    "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sdxl-prompt-encoder/real_weights.irpa"
  ]
}
```

Reply:
Hmm yeah, this is probably easier to decode/update